package.json

```diff
@@ -241,6 +241,7 @@
   "devDependencies": {
     "5to6-codemod": "5to6/5to6-codemod#v1.7.0",
     "babel-eslint": "6.1.2",
+    "babel-plugin-transform-builtin-extend": "1.1.2",
```

Comment:
We need to be sure to update the shrinkwrap when adding a dependency.

Reply:
See #19746

Reply:
Ahhhhhhhhhhhhhhhhhhh, sorry about that. Thanks for fixing it for me 🙇🏻